deps/ccommon/rust/cc_binding/build.rs

```diff
@@ -3,57 +3,64 @@ extern crate bindgen;
 extern crate failure;
 
 use std::env;
+use std::ffi::OsString;
 use std::fs;
 use std::io;
```

Comment:
You can combine this `use` with the one for `std::io::BufReader` eg: `use std::io::{self, BufReader}`

I agree with leaving the one including `prelude::*` on its own line.